index.bs

```diff
@@ -1216,7 +1216,7 @@ partial interface MLGraphBuilder {
   </summary>
   <div class=algorithm-steps>
     1. [=Assert=]: |op| is one of "argMin", "argMax".
-    1. If |options|.{{MLArgMinMaxOptions/axes}} [=map/exists=], if any of its elements is not in [=the range=] 0 to |input|'s [=MLOperand/rank=], exclusive, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
+    1. If |options|.{{MLArgMinMaxOptions/axes}} [=map/exists=], if any of its elements is not in [=the range=] 0 to |input|'s [=MLOperand/rank=], exclusive, then [=exception/throw=] a {{TypeError}}.
```

Comment:
This is checking whether the values are within a given range (suggesting perhaps `RangeError`), but since this is checking the values _within a list_ you could argue that `DataError` is more appropriate. This is further complicated by the fact that the range in question is dependent on the rank of the `MLOperand`. I've put it as `TypeError` for now, but am open to changing it. Any thoughts?

Reply:
`RangeError` seems more meaningful to me, because it indicates the axis value is illegal for the operation to access the operand's dimensions.